mzLib/MassSpectrometry/BaseClasses/FragmentationTerminus.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MassSpectrometry
+{
+    public enum FragmentationTerminus
+    {
+        Both, //N- and C-terminus
+        N, //N-terminus only
+        C, //C-terminus only
+        None, //used for internal fragments, could be used for top down intact mass?
+        FivePrime, // 5' for NucleicAcids
```

Comment:
Do we need a "both" for nucleic acids?

Reply:
Both was already in this enum from proteins. Unsure if it will be used/need for RNA